server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -1017,6 +1017,18 @@ public MeasurementSchema getSeriesSchema(PartialPath device, String measurement)
     return mtree.getChildNodePathInNextLevel(path);
   }
 
+  /**
+   * Get child node in the next level of the given path.
+   *
+   * <p>e.g., MTree has [root.sg1.d1.s1, root.sg1.d1.s2, root.sg1.d2.s1] given path = root.sg1,
+   * return [d1, d2] given path = root.sg.d1 return [s1,s2]
+   *
```

Comment:
this add paremater comment..
so @parameter path ....

Reply:
Thank you for this advice，I will adjust accordingly